modules/ttdBidAdapter.js

```diff
@@ -121,12 +120,12 @@ function getUser(bidderRequest) {
     utils.deepSetValue(user, 'ext.consent', bidderRequest.gdprConsent.consentString);
   }
 
-  if (utils.isStr(utils.deepAccess(bidderRequest, 'bids.0.userId.tdid'))) {
-    user.buyeruid = bidderRequest.bids[0].userId.tdid;
+  if (utils.isStr(utils.deepAccess(bidderRequest, 'bids.0.userIdAsEids.tdid'))) {
```

Comment:
I don't think you want to touch this. `userIdAsEids`, when it exists, is an array; `userIdAsEids.tdid` will always be undefined, because arrays do not have a `tdid` property.